cluster-autoscaler/FAQ.md

```diff
@@ -544,6 +559,34 @@ For example:
 
 For a complete list of the feature gates and their default values per Kubernetes versions, refer to the [Feature Gates documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
 
+### How can I use ProvisioningRequest to run batch workloads
+
+ProvisioningRequest (or ProvReq) is a new namespaced Custom Resource that allows you to ask CA for capacity for groups of pods. For a detailed explanation of the ProvisioningRequest API, please refer to the
+[proposal](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/proposals/provisioning-request.md).
```

Comment:
Is that up to date? IIRC we've made some changes from the original design during the implementation?

Reply:
I think implementation details are not correct, but the API didn't change except of suffix, so I think it's ok to refer this proposal for detailed explanation of the ProvisioningRequest API. 
I added "original proposal" to emphasise that some other details of the proposal could be outdated.